lib/LMSDocuments/LMSTcpdfInvoice.php

```diff
@@ -849,6 +859,7 @@ public function invoice_body_standard() {
 		$this->invoice_dates();
 		$this->invoice_expositor();
 		$this->invoice_footnote();
+		if (ConfigHelper::checkValue(ConfigHelper::getConfig('invoices.qr2pay', false)) AND !isset($this->data['rebate'])) $this->invoice_qr2pay_code();
```

Comment:
Prośba o poprawienie na:
```php
if (ConfigHelper::checkConfig('invoices.qr2pay') && !isset($this->data['rebate']))
    $this->invoice_qr2pay_code();
```

ConfigHelper::checkConfig() robi od razu co trzeba - zakłada niejawną wartość 'false'.

Reply:
Następna sprawa: co oznacza uzależnienie od rebate?

Reply:
Dobra z rebate już rozumiem - chodzi o to, żeby faktura była obciążeniem.